pennylane/debugging.py

```diff
@@ -105,3 +109,90 @@ def get_snapshots(*args, **kwargs):
         return dbg.snapshots
 
     return get_snapshots
+
+
+class PLDB(pdb.Pdb):
+    """Custom debugging class integrated with Pdb."""
+
+    __active_dev = None
+
+    def __init__(self, *args, **kwargs):
+        """Initialize the debugger, and set custom prompt string."""
+        super().__init__(*args, **kwargs)
+        self.prompt = "[pldb]: "
```

Comment:
Will this be mutable, or could it be defined via property?

Reply:
It shouldn't be mutable, I can re-define it as a property. It should be internal anyways

Reply:
Fixed